service-base/src/test/java/org/eclipse/hono/service/AbstractProtocolAdapterBaseTest.java

```diff
@@ -401,7 +401,7 @@ public void testGetRegistrationAssertionFailsWith404ForNonExistingDevice(final V
 
         final Checkpoint assertion = ctx.checkpoint();
         // WHEN an assertion for a non-existing device is retrieved
-        adapter.getRegistrationAssertion("tenant", "non-existent", null)
+        adapter.getRegistrationAssertion("tenant", "non-existent", null, mock(SpanContext.class))
```

Comment:
same here
as long as we have the deprecated methods in the `RegistrationClient`, we will keep this test, I'd say ...

Reply:
`RegistrationClient` has no deprecated methods. Do you mean `AbstractProtocolAdapterBase`? yes, we can remove the usage in the above tests, when we remove the deprecated method from  `AbstractProtocolAdapterBase`.